packages/venia-ui/lib/context/catalog.js

```diff
@@ -0,0 +1,39 @@
+import React, { createContext, useMemo } from 'react';
+
+import catalogActions from '../actions/catalog';
+import { connect } from '../drivers';
+
+export const CatalogContext = createContext();
+
+const CatalogContextProvider = props => {
+    const { catalog: catalogState, children, updateCategories } = props;
+
+    const catalogApi = useMemo(
+        () => ({
+            updateCategories
```

Comment:
Do we want to add the other actions like `addFilter` and `removeFilter`? I think we should make sure that each of these new contexts exports every action available and then in a subsequent PR we can clean up unused ones.

Reply:
```js
import * as actions from '../actions/catalog';
const catalogApi = useMemo(
  () => ({
    ...actions
  })
);
```

or something...